core/src/main/java/io/squashql/query/database/SqlUtils.java

```diff
@@ -25,10 +27,20 @@ public static String getFieldFullName(String store, String name) {
     return store == null ? name : store + '.' + name;
   }
 
-  public static String getFieldFullName(TypedField field) {
+  public static String getFieldFullName(TableTypedField field) {
     return field.store() == null ? field.name() : field.store() + '.' + field.name();
   }
 
+  public static String expression(TypedField f) {
```

Comment:
since TypedField are internals maybe we can leverage polymophirsm and add a method expression ?